subcommittees/mentoring/roles.md

```diff
@@ -0,0 +1,111 @@
+# Mentoring Subcommittee Roles Roles and Responsibilities
+
+## Roles
+
+The members of the Mentoring Subcommittee may have particular roles within the 
+functioning of the subcommittee. These roles include:
+
+* [Chair](#chair)
+* [Vice Chair](#vicechair)
+* [Secretary](#secretary)
+
+### Chair
+
+The Chair is meant to organize and facilitate the activities of the Committee.
+The Chair agrees to:
+
+1.  Organize/schedule meetings at the agreed-upon frequency.
+
+2.  Ensure that an agenda is planned and circulated in a timely
+    fashion.
+
+3.  Lead meetings fairly and transparently (or delegate meeting
+    leadership when absent).
+
+4.  Serve as the primary liaison for communication between the
+    community and the Mentoring Subcommittee members.
+
+### Vice Chair
+
+The Vice Chair is meant to support the duties of the chair. In particular, the 
+Vice Chair should adopt the responsibilities of the Chair when the Chair is 
+temporarily unable to perform their duties (e.g., absent from a meeting, 
+stepping down during a vote temporarily due to conflict of interest). The Vice
+Chair agrees to:
+
+1.  Adopt the duties and authority of the Chair at the request of the Chair or
+    by vote of the Committee.
+
+2.  Act as the primary liaison between the Steering Committee and the Subcommittee.
```

Comment:
I would not include 2. The liaison may or may not be the vice chair, I don´t think this should be a given.